Makefile

```diff
@@ -424,7 +424,7 @@ fv fv/latency.log fv/data-races.log: $(REMOTE_DEPS) image-test bin/iptables-lock
 	fi
 
 fv-bpf:
-	$(MAKE) fv FELIX_FV_ENABLE_BPF=true
+	$(MAKE) fv FELIX_FV_ENABLE_BPF=true GINKGO_FOCUS=BPF-SAFE
```

Comment:
Can this be overridden from the command line?

Reply:
Good point, I'll fix that.